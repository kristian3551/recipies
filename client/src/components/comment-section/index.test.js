import React from 'react';
import renderer from 'react-test-renderer';
import CommentSection from './index';

describe('<CommentSection/>', () => {
    it('should render right things', () => {
        const component = renderer.create(<CommentSection comments={
            [{
                "createdAt": "2021-04-09T06:10:11.734Z",
                "_id": "60700e33a9e9c50e20f433a6",
                "content": "I enjoyed this recipe!",
                "author": "kristian01",
                "__v": 0
              }, {
                "createdAt": "2021-04-09T06:10:11.734Z",
                "_id": "60700e33a9e9c50e20f433a6",
                "content": "I enjoyed this recipe!",
                "author": "kristian01",
                "__v": 0
              }]
        }/>);
        expect(component.toJSON()).toMatchSnapshot();
    })
});

