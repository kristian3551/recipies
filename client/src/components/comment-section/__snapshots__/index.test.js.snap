// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CommentSection/> should render right things 1`] = `
<div
  className="comment_block"
>
  <div
    className="create_new_comment"
  >
    <div
      className="input_comment"
    >
      <input
        onChange={[Function]}
        placeholder="Join the conversation.."
        type="text"
      />
      <button
        className="btn btn-outline-light"
        onClick={[Function]}
        style={
          Object {
            "float": "right",
          }
        }
        type="button"
      >
        Submit comment
      </button>
    </div>
  </div>
  <div
    className="new_comment"
  >
    <ul
      className="user_comment"
    >
      <div
        className="comment_body"
      >
        <p>
          I enjoyed this recipe!
        </p>
      </div>
      <div
        className="comment_toolbar"
      >
        <div
          className="comment_details"
        >
          <ul>
            <li
              style={
                Object {
                  "color": "black",
                }
              }
            >
              <i
                className="fa fa-clock-o"
                style={
                  Object {
                    "color": "black",
                  }
                }
              />
               
              9
              :
              10
            </li>
            <li
              style={
                Object {
                  "color": "black",
                }
              }
            >
              <i
                className="fa fa-calendar"
              />
              9
              /
              4
              /
              2021
            </li>
            <li
              style={
                Object {
                  "color": "black",
                }
              }
            >
              <i
                className="fa fa-pencil"
              />
               
              <span
                className="user"
              >
                kristian01
              </span>
            </li>
          </ul>
        </div>
      </div>
    </ul>
  </div>
  <div
    className="new_comment"
  >
    <ul
      className="user_comment"
    >
      <div
        className="comment_body"
      >
        <p>
          I enjoyed this recipe!
        </p>
      </div>
      <div
        className="comment_toolbar"
      >
        <div
          className="comment_details"
        >
          <ul>
            <li
              style={
                Object {
                  "color": "black",
                }
              }
            >
              <i
                className="fa fa-clock-o"
                style={
                  Object {
                    "color": "black",
                  }
                }
              />
               
              9
              :
              10
            </li>
            <li
              style={
                Object {
                  "color": "black",
                }
              }
            >
              <i
                className="fa fa-calendar"
              />
              9
              /
              4
              /
              2021
            </li>
            <li
              style={
                Object {
                  "color": "black",
                }
              }
            >
              <i
                className="fa fa-pencil"
              />
               
              <span
                className="user"
              >
                kristian01
              </span>
            </li>
          </ul>
        </div>
      </div>
    </ul>
  </div>
</div>
`;
